### ISSUE TITLE
Dropdown power menu option is not readable

### ISSUE THREAD

**Issue by mmyskiv11** [vinceliuice/Graphite-gtk-theme#125]
I am using the black variant of the theme (./install.sh -c dark --libadwaita --tweaks rimless normal black)

![20240112_045225.jpg](https://github.com/vinceliuice/Graphite-gtk-theme/assets/139520250/a989772c-c26c-40fa-b815-05518292686f)

**Comment by dominik-muc** [comment]
Same issue with all dropdown menus.
![image](https://github.com/vinceliuice/Graphite-gtk-theme/assets/72618648/3c04e8e3-f399-4586-b9b7-adc2aa663b36)
![image](https://github.com/vinceliuice/Graphite-gtk-theme/assets/72618648/bdc0f70b-4753-489e-a42a-90eb07f0a782)

**Comment by vinceliuice** [comment]
Did you installed GDM theme ?

**Comment by dominik-muc** [comment]
@vinceliuice Yes, but it works in GDM, however the color is white, even though I've installed blue variation.
Commands I have used:
`$ ./install.sh -c dark -t blue -l --tweaks normal`
`# ./install.sh -c dark -t blue -l -g --tweaks normal`
I tested on my laptop, that without installing GDM theme, dropdown menus work as should.
Both my devices are on Arch Linux and I installed all needed dependencies.
